python/rascaline/rascaline/calculator_base.py

```diff
@@ -197,100 +197,104 @@ def compute(
         r"""Runs a calculation with this calculator on the given ``systems``.
 
         :param systems: single system or list of systems on which to run the
-            calculation. The systems will automatically be wrapped into
-            compatible classes (using :py:func:`rascaline.systems.wrap_system`).
-            Multiple types of systems are supported, see the documentation of
+            calculation. The systems will automatically be wrapped into compatible
+            classes (using :py:func:`rascaline.systems.wrap_system`). Multiple types of
+            systems are supported, see the documentation of
             :py:class:`rascaline.IntoSystem` to get the full list.
 
         :param use_native_system: If :py:obj:`True` (this is the default), copy data
-            from the ``systems`` into Rust ``SimpleSystem``. This can be a lot
-            faster than having to cross the FFI boundary often when accessing
-            the neighbor list. Otherwise the Python neighbor list is used.
+            from the ``systems`` into Rust ``SimpleSystem``. This can be a lot faster
+            than having to cross the FFI boundary often when accessing the neighbor
+            list. Otherwise the Python neighbor list is used.
 
-        :param gradients: List of gradients to compute. If this is :py:obj:`None` or
-            an empty list ``[]``, no gradients are computed. Gradients are
-            stored inside the different blocks, and can be accessed with
-            ``descriptor.block(...).gradient(<parameter>)``, where
-            ``<parameter>`` is ``"positions"`` or ``"cell"``. The following
-            gradients are available:
+        :param gradients: List of gradients to compute. If this is :py:obj:`None` or an
+            empty list ``[]``, no gradients are computed. Gradients are stored inside
+            the different blocks, and can be accessed with
+            ``descriptor.block(...).gradient(<parameter>)``, where ``<parameter>`` is
+            ``"positions"``, ``"cell"`` or ``"strain"``. The following gradients are
```

Comment:
Since the gradients user want to calculate for virials are gradients wrt to strain we should maybe swap the order of mentioning to: 

```suggestion
            ``"positions"``, ``"strain"`` or ``"cell"``. The following gradients are
```

this also reflects how to list them below.